### PR TITLE
Make AIX known by bootstrap

### PR DESCRIPTION
**Pull request by bzEq**
Use `x.py` to build rustc on AIX directly is failing
```
unknown OS type: AIX
Build completed unsuccessfully in 0:00:00
```
If kernel is `AIX`, we should return default triple `powerpc64-ibm-aix` for current rustc.

### CODE REVIEW

File: src/bootstrap/bootstrap.py [hunk 1]
```diff
@@ -312,6 +312,8 @@ def default_build_triple(verbose):
         # non-standard string (e.g. gnuwin32 tools returns `windows32`). In
         # these cases, fall back to using sys.platform.
         return 'x86_64-pc-windows-msvc'
+    elif kernel == 'AIX':
```
Comment (Nilstrieb) [inline src/bootstrap/bootstrap.py:315]: Would it make more sense to add AIX to the kerneltype map above?

Comment (bzEq) [reply src/bootstrap/bootstrap.py:315]: I've added comment to describe the reason :).

**Comment by Nilstrieb** [comment]
r? bootstrap

**Comment by albertlarsan68** [comment]
Flipping back to the original assignee...
r? @Mark-Simulacrum

**Pull request merged**